### PR TITLE
Added Logging+ Extension

### PR DESCRIPTION
**Pull request by Shawn-E**

### CODE REVIEW

**Comment by Itz-Hyperz** [comment]
It's a security flaw to include that license key in your `onStart` event. Should not publicly log tokens anywhere.

File: Logging+/logging+.js [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+  faxstore.emit('CreateAuditLog', userObject.id, 'New User Account', `Account created at <t:${date}> | Login Type: ${serviceType}`);
+});
+
+faxstore.on('onStart', function (licenseKey, siteDomain) {
+
+  const date = Date.now();
+
+  const embed = new MessageEmbed()
+    .setTitle(`Faxstore Started`)
+    .setDescription(`**License Key:** ||\`${licenseKey}\`||`)
```
Comment (Itz-Hyperz) [inline Logging+/logging+.js]: Remove this.

Comment (Itz-Hyperz) [reply Logging+/logging+.js]: Just line 87.

Comment (Shawn-E) [reply Logging+/logging+.js]: Yep, thank you for pointing that out i just removed it

**Review by FAXES — COMMENTED** [review]
Awaiting testing

**Pull request merged**